packages/react-server-cli/src/compileClient.js

```diff
@@ -165,6 +166,13 @@ const packageCodeForBrowser = (entrypoints, outputDir, outputUrl, hot, minify, l
 				},
 			],
 		},
+		resolve: {
+			alias: {
+				// These need to be singletons.
+				"react"        : callerDependency("react"),
```

Comment:
Iiuc, if you want to break all your react components into modules that export the component, actions, stores &c, this means you'll all agree on the same react and react-server.  Should we also resolve `q`? `reflux`? `react-dom`?